pyproject.toml

```diff
@@ -60,10 +60,14 @@ skip_glob = ["tests/fixtures/common/collections*"] # Skip ansible content due to
 
 [tool.pylint]
     [tool.pylint.master]
-    ignore="tm_tokenize"
+    ignore="tm_tokenize" # virtually vendored and shouldn't be linted as such
 
     [tool.pylint.messages_control]
-    disable = "C0330, C0326"
+    disable = [
+      "duplicate-code",
+      "unsubscriptable-object",
+      "fixme"
```

Comment:
This should have trailing commas on every line.

Reply:
this file isn't modified in the Pr anymore, I think this was from the original implementation, 

I think it was covered by: https://github.com/ansible/ansible-navigator/pull/800

Reply:
Yep.